hbp_knowledge/wardyn2015.bel

```diff
@@ -0,0 +1,481 @@
+# DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
+
+##############
+# Statements #
+##############
+
+SET Citation = {"PubMed", "26551702"}
+
+SET Evidence = "Among the most studied Nrf2-target genes are NAD(P)H
+dehydrogenase quinone 1 (NQO1), heme oxygenase-1 (HO-
+1), γ -glutamyl cysteine ligase modulatory subunit (GCLM),
+the catalytic subunit (GCLC) and ferritin, which function
+to maintain a reducing environment within the cell [2–4]."
+#Nrf2 is NFE2L2
+#target as transcription factor
+tscript(p(HGNC:NFE2L2)) reg r(HGNC:NQO1)
+tscript(p(HGNC:NFE2L2)) reg r(FPLX:HMOX)
+tscript(p(HGNC:NFE2L2)) reg r(HGNC:GCLM)
+tscript(p(HGNC:NFE2L2)) reg r(HGNC:GCLC)
+tscript(p(HGNC:NFE2L2)) reg r(HGNCGENEFAMILY:"Ferritin subunits")
+
+SET Evidence = "Moreover, Nrf2 also plays a prominent role in
+orchestrating glucose metabolism by mediating transcription
+of components of the pentose phosphate pathway [7]."
+p(HGNC:NFE2L2) -- bp(GO:"glucose metabolic process")
+p(HGNC:NFE2L2) reg bp(GO:"pentose-phosphate shunt")
+
+SET Evidence = "The best
+characterized mechanism of Nrf2 regulation is mediated
+by interaction with the Kelch-like ECH-associated protein
+1 (Keap1)–Cullin3–Rbx1 complex, which mediates Nrf2
+ubiquitination and subsequent proteasomal degradation [8]."
+complex(p(HGNC:KEAP1),p(HGNC:CUL3),p(HGNC:RBX1)) reg p(HGNC:NFE2L2)
+complex(p(HGNC:KEAP1),p(HGNC:CUL3),p(HGNC:RBX1)) -> p(HGNC:NFE2L2,pmod(Ub))
+p(HGNC:NFE2L2,pmod(Ub)) -> deg(p(HGNC:NFE2L2))
+bp(GO:"proteasomal protein catabolic process") -> deg(p(HGNC:NFE2L2))
+
+SET Evidence = " Oxidizing and
+electrophilic agents, such as free ROS and plant-derived
+phenolic compounds, among many others, can modify these
+cysteine residues [9]. This causes a conformational change in
+Keap1, preventing subsequent binding of newly synthesized
+Nrf2, which promptly accumulates in the nucleus [10]."
+a(CHEBI:"reactive oxygen species") -| complex(p(HGNC:KEAP1),p(HGNC:NFE2L2))
+a(CHEBI:"phenolic radical") -| complex(p(HGNC:KEAP1),p(HGNC:NFE2L2))
+complex(p(HGNC:KEAP1),p(HGNC:NFE2L2)) -| p(HGNC:NFE2L2,loc(MESH:"Cell Nucleus"))
+
+SET Evidence = " There,
+Nrf2 associates with small Maf proteins (sMaf) and binds to
+antioxidant responsive elements (ARE) in the promoters of
+its target genes."
+SET MeSHAnatomy = "Cell Nucleus"
+p(HGNC:NFE2L2,loc(MESH:"Cell Nucleus")) -> complex(p(FPLX:MAF),p(HGNC:NFE2L2))
+p(HGNC:NFE2L2,loc(MESH:"Cell Nucleus")) -> complex(g(MESH:"Antioxidant Response Elements"),p(HGNC:NFE2L2))
+UNSET MeSHAnatomy
+
+SET Evidence = "however, Nrf2 can either be
+degraded in the nucleus via the β-TrCP–GSK3β axis or
+alternatively it may translocate back to the cytoplasm where
+it is swiftly degraded by Keap1 [12,13]."
+p(HGNC:BTRC) -> deg(p(HGNC:NFE2L2))
+p(HGNC:GSK3B) -> deg(p(HGNC:NFE2L2))
+tloc(p(HGNC:NFE2L2), MESH:"Cell Nucleus", MESH:Cytosol) -> deg(p(HGNC:NFE2L2))
```

Comment:
please use fromLoc....toLoc to desribe translocation
check the whole document for that